client/my-sites/sidebar/sidebar.jsx

```diff
@@ -185,14 +187,14 @@ export class MySitesSidebar extends Component {
 		return (
 			<SidebarItem
 				tipTarget="menus"
-				label={ isEnabled( 'customer-home' ) ? translate( 'Home' ) : translate( 'Checklist' ) }
+				label={ isCustomerHomeEnabled ? translate( 'Home' ) : translate( 'Checklist' ) }
 				selected={ itemLinkMatches(
 					isEnabled( 'customer-home' ) ? [ '/home' ] : [ '/checklist' ],
 					path
 				) }
-				link={ isEnabled( 'customer-home' ) ? '/home' + siteSuffix : '/checklist' + siteSuffix }
+				link={ isCustomerHomeEnabled ? '/home' + siteSuffix : '/checklist' + siteSuffix }
 				onNavigate={ this.trackCustomerHomeClick }
-				materialIcon={ isEnabled( 'customer-home' ) ? 'home' : 'check_circle' }
+				materialIcon={ isCustomerHomeEnabled ? 'home' : 'check_circle' }
```

Comment:
What if you consolidate the props in a single one that is later expanded?

```js
const sidebarItemProps = isCustomerHomeEnabled
    ? {
        label: translate( 'Home' ),
        link: '/home' + siteSuffix,
        materialIcon: 'home',
    }
    : {
        label: translate( 'Checklist' ),
        link: '/checklist' + siteSuffix,
        materialIcon: 'check_circle',
    };

    return (
        <SidebarItem
            ...sidebarItemProps
            tipTarget="menus"
```

Reply:
Much better! I'll update that